examples-app/src/App.css

```diff
@@ -48,3 +48,40 @@
 .read-the-docs {
   color: #888;
 }
+@keyframes slideInLeft {
```

Comment:
Here and in the code should use "previous"/"next" instead of "left/right" and consider the `dir="rtl"` case (e.g. arabic calendar).